packages/components/SimpleCard/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import { LinkContainer } from 'react-router-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faTrash,
+    faPen
+} from '@fortawesome/free-solid-svg-icons';
+import 'bootstrap-css-only/css/bootstrap.css';
+
+class SimpleCard extends Component {
+
+    handleDelete = () => {
+        const confirmDeleteText = "Click OK if you wish to delete or cancel if you don't";
+        const cancelDelteText = "All is good, it was not deleted";
+        const confirmationPrompt = confirm(confirmDeleteText);
+        
+        if (confirmationPrompt) {
+            this.props.handleDelete ? this.props.handleDelete(this.props.id) : alert(cancelDelteText);
+        }
+    };
+
+    handleEdit = () => {
+        this.props.handleEdit(this.props.id);
+    }
+
+    showLinkPath = () => {
+        return this.props.showLinkPath(this.props.id) || '';
```

Comment:
Maybe worth implementing a refactor to simplify the logic later on when you are working on the `List`.
`showLinkPath` shouldn't be passed on here as it over-complicates things.
`to` functionality in the LinkContainer should be taking in `string` of a url - and redirecting to that. Not taking in a `showLinkPath` function.